### ISSUE TITLE
How well does this work?

### ISSUE THREAD

**Issue by vanquishedangel** [yarrick/turbomem#3]
Hi, I don't really have an issue but I am wondering how well this works as I am updating an old dell d430 and ordered a Robson chip. I am curious if this will boost it, or can be used to boost the dell. Thank you in advance.

**Comment by 3u128** [comment]
I've tested in ThinkPad T400 with a cheap SSD. This chip is really slow and I don't use it. But maybe you have some software or tools that use it.

**Comment by vanquishedangel** [comment]
> I've tested in ThinkPad T400 with a cheap SSD. This chip is really slow and I don't use it. But maybe you have some software or tools that use it.

 By chance. have any speed measurements?

The dell may benefit even though I put an ssd in it because the dell d430 uses a **ziff pata** connection for the hard drive and not sata, so an adapter was required from pata to m.2.  The ziff pata connection has a max speed of 85mbps on a good day in this old netbook. I have its old 4200 spining hard disk as a back up so it most likely would help there as well if I ever needed to go back to use it.

**Comment by yarrick** [comment]
This driver does not work properly. Also modern SSDs are much faster than this old one.

**Comment by vanquishedangel** [comment]
> This driver does not work properly. Also modern SSDs are much faster than this old one.

All good but I do not think you read my response completely as due to the "ziff Pata" connection, even with a modern SSD, the main hard disk will not ever be faster then 85mbps. However if they make an SSD that fits the mini pcie protocol like the robson chip, then you would also be correct.

**Comment by vanquishedangel** [comment]
So I received my Robson chip today and put it in the computer, I have some tests for you guys :)

/dev/sdb (robson Chip):
 Timing cached reads:   1396 MB in  2.00 seconds = 698.05 MB/sec
 Timing buffered disk reads:  56 MB in  3.05 seconds =  18.39 MB/sec
shawn@shawn-latituded430:~$ sudo hdparm -Tt /dev/sda

/dev/sda (main Hard drive SSD with a pata connection):
 Timing cached reads:   1408 MB in  2.00 seconds = 704.35 MB/sec
 Timing buffered disk reads: 252 MB in  3.02 seconds =  83.52 MB/sec
shawn@shawn-latituded430:~$ sudo hdparm -Tt /dev/sdc

/dev/sdc (swap space 2gig amtron PCMCIA card):
 Timing cached reads:     2 MB in  2.32 seconds = 882.25 kB/sec
 Timing buffered disk reads:   4 MB in  4.65 seconds = 879.92 kB/sec
shawn@shawn-latituded430:~$

**Comment by yarrick** [comment]
In that case it is not a robson chip but something else. Especially if it works like that without any special driver

**Comment by vanquishedangel** [comment]
I think you are correct, and the files I found on it were older then the ones on my computer (though initially it showed the drive as blank, I had to install an msdos library). On the chip was linux software (though may have been windows versions) and some windows. The dates were old (2007-2017) and I was not able to erase them. I formatted it and am using it as swap space currently, but I think this chip was dell fixing the robson chip issues and they made their own for a time.

Anyway, it will work as swap to alleviate bottle necks . Until I figure out dm-cache better.
